configure.py

```diff
@@ -1022,6 +1022,34 @@ def cc_macros(cc=None):
       key = lst[1]
       val = lst[2]
       k[key] = val
+
+  try:
+    p = subprocess.Popen(shlex.split('uname') + ['-v'],
+                         stdin=subprocess.PIPE,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE)
+  except OSError:
```

Comment:
Could someone check if other types of error/exception could happen in other environments when `uname` is not available? I tried with different command names that are def. not available and the catch/forget code works well